### ISSUE TITLE
SentrySwift - Error:: The operation couldn’t be completed. (Cocoa error 260.)

### ISSUE THREAD

**Issue by phuonglm86** [getsentry/sentry-swift#43]
Suddenly got it.
<img width="1180" alt="screen shot 2016-07-13 at 17 07 18" src="https://cloud.githubusercontent.com/assets/8994583/16799780/a0b9706a-491c-11e6-835a-e96a959fc2ea.png">

**Comment by phuonglm86** [comment]
No idea why getting Cocoa error 260 ?!

**Comment by joshdholtz** [comment]
@phuonglm86 Are you getting this error every time? Or just on first run?

**Comment by phuonglm86** [comment]
@joshdholtz: Getting it every time, yes.

**Comment by joshdholtz** [comment]
@phuonglm86 Looking into right now! Thanks for letting me know of this. Will hopefully have a fix for you soon 😊

**Comment by joshdholtz** [comment]
@phuonglm86 Not sure why yours showed `Cocoa Error 260` but I ended up changing this log message from an `error` to `debug`. `SentryClient` looks for that directory when it starts up to see if there are any saved crash reports it needs to send it. A Swift error is thrown if it can't find the directory so I was originally logging it as an error even though that directory may actually not exist. I changed it to a debug message instead so it won't show unless you turn on debug logging 😊
